### ISSUE TITLE
On calendar, hide times before 8:00 a.m. and after 6:00 p.m. 

### ISSUE THREAD

**Issue by smplater** [aih/FlatGov#60]
***Don't hide these timeframes if there is an event that occurs

**Comment by smplater** [comment]
![Image 2020-12-15 at 11 20 43 AM](https://user-images.githubusercontent.com/60105995/102242083-ae61ef80-3ec7-11eb-950d-3de9074c739e.png)

**Comment by leedavidr** [comment]
I don't think there will be an easy way to render the time view differently when events are present at an earlier or later time than 8 AM to 6 PM.  I will just set it for 7 AM to 7 PM for now to account for more events. We can adjust to 8 AM to 6 PM if that is preferred.

We're using fullcalendar js, which comes with some configuration properties. There could be an easy way to achieve this but it wasn't obvious. Example configurations: https://fullcalendar.io/docs/slotMinTime

There are also probably custom ways to achieve this, but I wanted to spend my time on more valuable items first, and I can come back to look deeper if prioritized

**Comment by aih** [comment]
@leedavidr I think we should prioritize data issues: scraping, organizing, saving. What we ultimately want to do with this kind of UI adjustment is likely to depend a lot on what the data looks like. Related to this, I think we should go through issues and add labels that would prioritize. It's good to have presentation issues documented, but more critical to the project goal to get the data in place. Thoughts?

**Comment by leedavidr** [comment]
@aih I agree, I was working on analyzing the data from feeds and wanted to make sure our data model supports everything the UI needs. This requirement was not supported by the library, so I wanted to know quickly if I needed to think about a custom calendar UI vs. save a lot of time by using an existing library

Re-visiting this issue after starting on the custom events API
We should be able to query the daterange for the events, and if the events for the daterange has an event outside the range, we can reinitialize the calendar
There may be slight visual delays and not optimal for system performance, but I think it could be possible to do this with current set up

Will keep low priority until main functionality is complete
